src/realm/index_string.cpp

```diff
@@ -28,6 +28,7 @@
 #include <realm/table.hpp>
 #include <realm/timestamp.hpp>
 #include <realm/column_integer.hpp>
+#include <realm/unicode.hpp>
```

Comment:
what is this for?

Reply:
`case_map`. This header was probably included implicitly before.